EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/attributeInput/AttributeInput.java

```diff
@@ -320,6 +371,14 @@ public static int getRandomIndex(Object[] inputArray) {
 		}
 
 		/**
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AJavadoc_Style_BP2016)